fidesctl/demo_resources/demo_data_uses.yml

```diff
@@ -0,0 +1,17 @@
+data_use:
+  - fides_key: third_party_sharing.legal_obligation.payroll
+    name: Payroll
+    description: Legally obliged sharing of payroll information
+    recipients: 
+    - HMRC
+    - IRS
+    - NYDTF
```

Comment:
Other feedback from team discussion today, providing more clearly modeled items for use in here could be beneficial (thinking `legal_basis` as a good example of including items in a default taxonomy, recipients could be tougher)

Possibly having a `generate data-use` (or `annotate`) command to help in building out required attributes

Having this become a rich attribute in relation to an eventual RoPA is that an engineer can run through modeling this in almost a one-stop manner

Reply:
Pushed an update for legal basis, still stewing about the cost/benefit of recipients and how to do that well if we want to pursue it

Reply:
Spent some time also looking at making `third_countries` more structurally defined using ISO 3166 country codes, fed from something [like this](https://restcountries.com/v2/all?fields=name,alpha3Code) but feel like it may be getting a little more complicated than we need to keep making progress towards our goal. Currently in a similar boat with recipients, where it might make sense to keep moving ahead for now as is and circle back with adding some more guiding structure and straight forward extensibility around these in a follow on issue (ideally also in 1.4)

Going to give it another once over tonight before asking for some reviews